### ISSUE TITLE
Is it possible to intercept hideOnClickOutside?

### ISSUE THREAD

**Issue by juona** [MrFrankel/ngx-popper#136]
Hi! The wrapper works great, thanks for your work. Here's an issue I'd like to solve: I open the popper using a button click. I would like the popper to remain open if I click the same button again and only close if I click anywhere else. How could I achieve that?

**Comment by tonysamperi** [comment]
![image](https://user-images.githubusercontent.com/5957244/93239978-d1271f80-f783-11ea-89c5-2f3b89f56d01.png)

@juona use the disabled attribute! When disabled, the ouside click won't close the popper!

By the way I just released ngx-popperjs@8.0.3 which now works with Angular8. Will upgrade to Angular 9 and 10 in the next few days...
Demo here => https://tonysamperi.github.io/ngx-popperjs

**Comment by juona** [comment]
Thanks, but unfortunately that's not what I wanted : ]

- outside click closes popover = GOOD
- same button click closes popover = BAD

I wish the popper to remain open if I click the same button that was used to open it in the first place. And close if clicked anywhere else.

**Comment by tonysamperi** [comment]
Yeah I had already understood what you need, but I hadn't noticed that the changing *disabled* attribute also hides the popover...
It seems that you need a very precise solution that you could achieve with a few tricks...
